src/rust/plugin-registry/tests/test_list_plugins.rs

```diff
@@ -0,0 +1,114 @@
+#![cfg(feature = "integration_tests")]
+
+use bytes::Bytes;
+use clap::Parser;
+use grapl_utils::future_ext::GraplFutureExt;
+use rust_proto::{
+    client_factory::{
+        build_grpc_client,
+        services::PluginRegistryClientConfig,
+    },
+    graplinc::grapl::api::plugin_registry::v1beta1::{
+        ListPluginsRequest,
+        PluginMetadata,
+        PluginType,
+    },
+};
+
+fn get_example_generator() -> Result<Bytes, std::io::Error> {
+    std::fs::read("/test-fixtures/example-generator").map(Bytes::from)
+}
+
+#[test_log::test(tokio::test)]
+async fn test_list_plugins() -> eyre::Result<()> {
```

Comment:
Can you add an edge case test for "a tenant id with no plugins"? should be just "build a tenant, add no generators/analyzers, call list_plugins"

Reply:
done